content/posts/newsletter/2019-12-03.md

```diff
@@ -0,0 +1,135 @@
+---
+title: XMPP in all languages! 03 Dec 2019
+save_as: 2019/12/newsletter-03-december/index.html
+url: 2019/12/newsletter-03-december/
+date: 2019-12-03 15:00
+author: nyco
+blog_id: newsletter
+category: Newsletter
+---
+
+## Intro 
+
+Welcome to the XMPP newsletter covering the month of November 2019.
+
+Help us sustain this as a community effort, which [process is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+## Articles 
+
+Edivaldo Brito has written in Portugese "[Como instalar o moderno cliente Jabber/XMPP Dino no Linux via Snap](https://www.edivaldobrito.com.br/cliente-jabber-xmpp-dino-no-linux/)" ("How to install the modern Jabber/XMPP Dino client on Linux via Snap").
+
+Last month, the XMPP newsletter has been translated by community members:
+* French: [Lettre d’information XMPP, 8 novembre 2019, sprints, IoT et le début de Twitter](https://linuxfr.org/news/lettre-d-information-xmpp-8-novembre-2019-sprints-iot-et-le-debut-de-twitter)
+* German: [XMPP Newsletter, 08. November 2019, Sprints, IoT und die Anfänge von Twitter](https://www.jabber.de/xmpp-newsletter-08-november-2019-sprints-iot-und-die-anfaenge-von-twitter/)
+* Spanish: [XMPP Newsletter, 08 Nov 2019, Sprints, IoT, y los primeros Twitter](https://www.jabber.de/es-xmpp-newsletter-08-nov-2019-sprints-iot-y-los-primeros-twitter/)
+
+
+We encourage you all to contribute to this translation effort!
+
+## Software releases
+          
+### Servers 
+
+
+The Ignite Realtime community has released:
+
+* [Openfire 4.4.4](https://discourse.igniterealtime.org/t/openfire-4-4-4-release/)
+* [HTTP File Upload plugin 1.1.2](https://discourse.igniterealtime.org/t/http-file-upload-plugin-1-1-2-released/) and [1.1.3](https://discourse.igniterealtime.org/t/http-file-upload-plugin-1-1-3-released/)
+* [inVerse Openfire plugin 5.0.5.1](https://discourse.igniterealtime.org/t/inverse-openfire-plugin-5-0-5-1-released/)
+
+ProcessOne has published bugfix release [ejabberd 19.09.1](https://blog.process-one.net/ejabberd-19-09-1/).
+
+[Metronome IM v3.13.0](https://github.com/maranda/metronome/releases/tag/v3.13.0) has been released, read the [changelog](https://metronome.im/changelogs/3_13).
+
+### Clients and applications 
+
+
+
+[Movim 0.16 – Cesco](https://nl.movim.eu/?node/pubsub.movim.eu/Movim/e10a8998-5e4d-4b52-9b9e-2f103b446d94) has been released, with Drawing and content sharing, Attachments, Chat improvements, and Chats, chatrooms list improvements, and Search, plus more.
+
+[gajim.org](https://gajim.org) features a new website. It is planned to publish a post about Gajim's development every month, starting with [October](https://gajim.org/post/2019-11-06-development-news-october/) and [November](https://gajim.org/post/2019-11-25-development-news-november/).
+
+Goffi has published his [SàT progress note 2019-W48](https://www.goffi.org/b/R5PeQUPBCSZWvabRNXtAE9/progress-note).
```

Comment:
Describe Goffi's role for people that don't know him.

Reply:
👍